lib/libutils/ext/include/confine_array_index.h

```diff
@@ -142,4 +142,11 @@ static inline size_t confine_array_index(size_t index, size_t size) {
   return safe_index;
 }
 #endif
+#ifdef __riscv
```

Comment:
Please add an empty line above. By the way, changes in this file should go into a separate commit.

Reply:
Ok, I'll analyze the Spectre V1 and put together a separate commit